grype/pkg/package_test.go

```diff
@@ -90,7 +90,7 @@ func TestNew_MetadataExtraction(t *testing.T) {
 						},
 						NamedSections: map[string]map[string]string{
 							"named-section": {
-								"named-section-key": "named-section-vaulue",
+								"named-section-key": "named-section-value",
```

Comment:
It's possible that this was intentionally misspelled, but it seemed unlikely.

Reply:
indeed unlikely!